src/storage/adapter/wasm.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use gloo_storage::LocalStorage;
+
+use super::StorageAdapter;
+
+/// The storage id.
+pub const STORAGE_ID: &str = "Wasm";
```

Comment:
Will this be the key in browsers local storage? If so, wouldn't it be better to call it something like `WasmWallet`?

Reply:
Right now its not used, couldnt verify how it looks yet in browsers. However its probably not needed at all. Ill remove it when i pr the full wasm bindings